demo/chess/index.js

```diff
@@ -17,65 +78,7 @@ class ChessPieceElement extends XElement {
 
   static template(html) {
     return ({ rank }) => {
-      return html`
-        <style>
```

Comment:
The bulk of the changes here are stop using inline `<style>` tags — this will throw a deprecation warning in future `1.x` releases (and be strictly forbidden in `2.x`).